scriptmodules/supplementary/runcommand/joy2key.py

```diff
@@ -30,6 +31,76 @@
 JS_EVENT_AXIS = 0x02
 JS_EVENT_INIT = 0x80
 
+RETROARCH_CFG = '/opt/retropie/configs/all/retroarch.cfg'
+DEFAULT_BTN_CODES = ['\n', '\t']
+
+def ini_get(key, cfg_file):
```

Comment:
Pretty much the same Regular Expressions found in inifuncs.sh with double-quotes `"` hardcoded.